<launch>
	<node pkg="visual_behavior" type="followboth_node" name="FollowBoth_Node" output="screen"/>
</launch>